.gitignore

```diff
@@ -1,4 +1,5 @@
 target
 __pycache__
 .vscode
-.idea
\ No newline at end of file
+.idea
+.DS_Store
```

Comment:
Don't add this change to the PR, these are entirely unrelated.

Reply:
> 

When I raise a pr, it gets attached automatically and pr gets two commits. So, rebased it and squashed it. What Should I do to make the commit ignore in this pr?

Reply:
Delete it, amend and rebase so it is a single commit

Reply:
> Delete it, amend and rebase so it is a single commit

okay.